### PR TITLE
Add ml_insights

### PR DESCRIPTION
**Pull request by philip-khor**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ml_insights/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "ml_insights" %}
```
Comment (synapticarbors) [inline recipes/ml_insights/meta.yaml:1]: I think we generally try to use the same name as on pypi, so this should be named `ml-insights`, but I can't find the official documentation/recommendation. @conda-forge/staged-recipes, can anyone remind me if this is correct?

File: recipes/ml_insights/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://ml-insights.readthedocs.io/en/latest/
+  summary: Package to calibrate and understand ML Models
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/ml_insights/meta.yaml:41]: add:

```
dev_url: https://github.com/numeristical/introspective
```

File: recipes/ml_insights/meta.yaml (hunk 1 above)
Comment (philip-khor) [reply recipes/ml_insights/meta.yaml]: in any case I've changed the name in the recipe and the folder name to ml-insights as suggested

Comment (ocefpaf) [reply recipes/ml_insights/meta.yaml]: @synapticarbors is correct, we try to use the same name. However, pip handles both _ and - as "the name" when installing. To mimick that we can use multiple outputs. Take a look at the pre-commit recipe if you want to go that route.

**Pull request merged**